index.html

```diff
@@ -60,223 +73,256 @@ <h3>Tonic</h3>
+                <ul class="social-media-icons no-bullets">
+                    <li>
+                        <a href="#" target="_blank"><i class="fa-solid fa-hand-peace fa-lg"></i></a>
+                    </li>
+                    <li><a href="https://twitter.com/lmunozm" target="_blank"><i
+                                class="fa-brands fa-twitter fa-lg"></i></a>
+                    </li>
+                    <li><a href="https://www.linkedin.com/in/luis-munoz-manriquez/" target="_blank"><i
+                                class="fa-brands fa-linkedin-in fa-lg"></i></a>
+                    </li>
+                    <li><a href="https://github.com/lmunozm1702" target="_blank"><i
+                                class="fa-brands fa-github fa-lg"></i></a>
+                    </li>
+                    <li><a href="https://medium.com/@lmunozm1702" target="_blank"><i
+                                class="fa-brands fa-medium fa-lg"></i></a>
+                    </li>
+                </ul>
+                <div><button class="see-project-button">Get my resume</button></div>
+            </main>
+            <aside class="about-myself-aside">
+                <div class="code-experience">
+                    <div class="list-header">
+                        <ul class="experience-nav">
+                            <li>Languages</li>
+                            <li><i class="fa-solid fa-angle-down"></i></i></li>
+                        </ul>
+                    </div>
+                    <div>
+                        <ul class="experience-list">
+                            <li class="experience-item">
+                                <div class="experience-img">
+                                    <img src="assets/images/javascript.png" alt="javaScript logo">
+                                </div>
+                                <div>
+                                    <p class="languaje-item">JavaScript</p>
+                                </div>
+                            </li>
+                            <li class="experience-item">
+                                <div class="experience-img">
+                                    <img src="assets/images/html5.png" alt="HTML-5 logo">
+                                </div>
+                                <div>
+                                    <p class="languaje-item">HTML</p>
+                                </div>
+                            </li>
+                            <li class="experience-item">
+                                <div class="experience-img">
+                                    <img src="assets/images/css.png" alt="CSS logo">
+                                </div>
+                                <div>
+                                    <p class="languaje-item">CSS</p>
+                                </div>
+                            </li>
+                        </ul>
+                    </div>
```

Comment:
[OPTIONAL] From the figma design, you can see that it is only the language sections which has been extended or open to show the languages available whiles the other two parts (Frameworks and Skills) are closed. I can see from your design that you opened those two and added some icons. You also changed the arrows on the right as shown in the image below. It is good to be innovative and creative, :man_artist:  but the purpose of this exercise is to conform to the dimensions of the figma design as strictly as possible. :disappointed_relieved:  You will get more opportunities to be creative in subsequent projects, but for now Kindly revert this changes back to the original as depicted on the figma designs. :pleading_face: 

Your design             |  Figma design
:-------------------------:|:-------------------------:
![Screenshot from 2022-09-29 23-07-17](https://user-images.githubusercontent.com/65233860/193158449-e2baae57-7441-4c09-9a1f-20b008cdc1e4.png)  |  ![Screenshot from 2022-09-29 23-07-43](https://user-images.githubusercontent.com/65233860/193158481-e98fd072-b1e5-4871-aa0e-19226a03d48e.png)

Reply:
Hi!! yes, but the instructions says "don't use javascript yet", and because I let "all opened" the lists! Thank's a lot!!

Reply:
> Hi!! yes, but the instructions says "don't use javascript yet", and because I let "all opened" the lists! Thank's a lot!!

I undestand you perfectly well, @lmunozm1702. That is why it should be just like in the design,same design, same arrows etc. 

![Screenshot from 2022-09-30 06-41-23](https://user-images.githubusercontent.com/65233860/193207132-37a207cb-1a8b-4091-8303-94e97b1dd423.png)